Assets/Scripts/Managers/HighlightManager.cs

```diff
@@ -131,6 +148,29 @@ private void ResetSelection(GameState gameState)
                 }
 
             }
+            // temp implementation
+            if (selectedPlayer == null)
```

Comment:
Similarly, implement or remove this. If something is blocking you here, document it somewhere.

Reply:
seed

Reply:
?

Reply:
Incomplete; ref incomplete elsewhere